index.php

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+     <head> 	
+     <title> Login Page </title> 
+     </head>
+     <body>
+	
+	<center><h1>Login Page</h1></center>
+	<div align="center" border>
+	<form action="login.php" method="post">
+	<label for="Email">Username:</label>
+  	<input type="email" name="email" id="Email" required><br><br>
```

Comment:
```suggestion
  	<input type="email" name="email" id="email" required><br><br>
```
inconsistent identifier naming convention